details/operator-sdk/controllers/astraconnector_controller.go

```
@@ -154,8 +154,11 @@ func (r *AstraConnectorController) Reconcile(ctx context.Context, req ctrl.Reque
 	}
 
 	// POST/PUT Managed Cluster
-	_ = r.updateAstraConnectorStatus(ctx, astraConnector, natsSyncClientStatus)
-	return ctrl.Result{RequeueAfter: time.Minute * conf.Config.ErrorTimeout()}, nil
+	err = r.updateAstraConnectorStatus(ctx, astraConnector, natsSyncClientStatus)
+	if err != nil {
```

We can ignore this error, this is from updating astra connector status